packages/eslint-plugin-pf-codemods/test/rules/v5/menu-search-rename.js

```diff
@@ -0,0 +1,42 @@
+const ruleTester = require("../../ruletester");
+const rule = require("../../../lib/rules/v5/menu-search-rename");
+
+ruleTester.run("menu-search-rename", rule, {
+  valid: [
+    {
+      code: `import { MenuSearchInput, MenuSearch } from '@patternfly/react-core'; <MenuSearch><MenuSearchInput /></MenuSearch>`,
+    },
+    // No @patternfly/react-core import
+    { code: `<MenuInput />` },
+  ],
+  invalid: [
+    {
+      code: `import { MenuInput } from '@patternfly/react-core'; <MenuInput><div /></MenuInput>`,
+      output: `import { MenuInput, MenuSearch, MenuSearchInput } from '@patternfly/react-core'; <MenuSearch><MenuSearchInput><div /></MenuSearchInput></MenuSearch>`,
```

Comment:
Only thing is that I feel like we should replace MenuInput in the output rather than keeping it and only adding MenuSearchInput.